src/data/boundary.json

```diff
@@ -87,26 +87,5 @@
       }
     ]
   },
-  "navigationHeaderItems": [
-    {
-      "label": "Overview",
-      "path": "/boundary",
-      "id": "overview"
-    },
-    {
-      "label": "Docs",
-      "path": "/boundary/docs",
-      "id": "docs"
-    },
-    {
-      "label": "API",
-      "path": "/boundary/api-docs",
-      "id": "api-docs"
-    },
-    {
-      "label": "Install",
-      "path": "/boundary/downloads",
-      "id": "downloads"
-    }
-  ]
+  "navigationHeaderItems": {}
```

Comment:
Leaving one comment for all of the `src/data` JSON files to explain this: setting this to an empty object makes it so that the new `NavigationHeader` does not break for non-beta products we haven't configured yet. This'll give us some time to figure out what we want to do for these in the interim.